### DISCUSSION TITLE
Sort pages on an arbitrary order using taxonomy

### DISCUSSION THREAD

**Discussion by mbursa** [bolt/core#1291] [Ideas / Feature requests]
Bolt 3 docs describes the following feature to sort content-type entries in a user-defined order:

https://docs.bolt.cm/3.7/howto/sortingorder-in-contenttypes

I do not see the functionality in the recent (rc14) version of Bolt 4 nor I do see it on the roadmap. Is it planned to be supported or does it work in a different way?

Thanks.

**Comment by bobdenotter** [comment]
Hi, 

No, this feature is not implemented yet. I'll add a `4.1` milestone for it.
